src/lib.rs

```diff
@@ -306,7 +306,11 @@ pub use error::TemporalError;
 
 #[cfg(feature = "sys")]
 #[doc(inline)]
-pub use sys::Temporal;
+pub use sys::{Temporal, DefaultHostSystem};
+
+
+#[doc(inline)]
+pub use host::EmptyHostSystem;
```

Comment:
issue: we should not to export this type.

It's mostly meant as a test type for documentation purposes. Even more so, it definitely shouldn't be in the lib re-export